.nvmrc

```
@@ -0,0 +1 @@
+16
```

```suggestion
18
```

I can’t see any dependencies here that would be incompatible with 18.

---

Thanks, updated.